### PR TITLE
[install] Complete pip section

### PR DESCRIPTION
**Pull request by valeriocos**
This PR completes the PIP section by adding the info needed to run and configure GrimoireLab. A PR to GrimoireLab repo has been submitted to include the menu.yml file and the setup.cfg for a local execution within the default-grimoirelab-settings folder.

Related to:
- https://github.com/jsmanrique/grimoirelab-user-guide/pull/17#issue-353341667
- https://github.com/chaoss/grimoirelab/pull/256

### CODE REVIEW

File: docs/install/pip.md
Comment (GeorgLink) [inline docs/install/pip.md]: How about adding this to the prerequisites section above?

**Review by GeorgLink — APPROVED** [review]
This looks great!

I was wondering if the prerequisites should be grouped together at the top -- that's my only comment.

File: docs/install/pip.md
Comment (valeriocos) [reply docs/install/pip.md]: Done! feel free to merge if the new version is OK for you.

**Comment by valeriocos** [comment]
@GeorgLink is there any blocker to not merge this PR?

**Comment by GeorgLink** [comment]
I don't think so

**Comment by valeriocos** [comment]
good! can you merge it, please?

**Comment by GeorgLink** [comment]
I had to switch to desktop. I didn't see a merge button on the Android app.

**Pull request merged**
